src/annotator/anchoring/pdf.js

```diff
@@ -377,19 +406,23 @@ function prioritizePages(position) {
     return result;
   }
 
-  return findPage(position.start).then(({ index }) => sortPages(index));
+  return findPage(position).then(({ index }) => sortPages(index));
 }
 
 /**
  * Anchor a set of selectors to a DOM Range.
  *
  * @param {HTMLElement} root
- * @param {Array} selectors - Selector objects to anchor
+ * @param {Selector[]} selectors - Selector objects to anchor
  * @return {Promise<Range>}
  */
 export function anchor(root, selectors) {
-  const position = selectors.find(s => s.type === 'TextPositionSelector');
-  const quote = selectors.find(s => s.type === 'TextQuoteSelector');
+  const position = /** @type {TextPositionSelector|undefined} */ (selectors.find(
+    s => s.type === 'TextPositionSelector'
+  ));
+  const quote = /** @type {TextQuoteSelector|undefined} */ (selectors.find(
+    s => s.type === 'TextQuoteSelector'
+  ));
```

Comment:
TypeScript is not able to figure out that `selectors.find(...)` implies the result is a certain type of selector so I had to annotate the expression.